### PR TITLE
[Storage] Split TIMEOUT constant

### PR DESCRIPTION
**Pull request by rakshith91**

### CODE REVIEW

**Comment by adxsdk6** [comment]
Can one of the admins verify this patch?

File: sdk/storage/azure-storage-blob/azure/storage/blob/_shared/base_client.py [hunk 1]
```diff
@@ -210,7 +210,9 @@ def _create_pipeline(self, credential, **kwargs):
             return config, kwargs["_pipeline"]
         config.transport = kwargs.get("transport")  # type: ignore
         if "connection_timeout" not in kwargs:
-            kwargs["connection_timeout"] = DEFAULT_SOCKET_TIMEOUT
+            kwargs["connection_timeout"] = CONNECTION_TIMEOUT
+        if "read_timeout" not in kwargs:
```
Comment (lmazuel) [inline sdk/storage/azure-storage-blob/azure/storage/blob/_shared/base_client.py]: Detail:
```python
if "read_timeout" not in kwargs:
    kwargs["read_timeout"] = READ_TIMEOUT
```
is the same thing as:
```python
kwargs.setdefault("read_timeout", READ_TIMEOUT)
```

File: sdk/storage/azure-storage-blob/azure/storage/blob/_shared/constants.py [hunk 2]
```diff
@@ -11,14 +11,16 @@
...
+CONNECTION_TIMEOUT = 20
+READ_TIMEOUT = 20
 
 # for python 3.5+, there was a change to the definition of the socket timeout (as far as socket.sendall is concerned)
 # The socket timeout is now the maximum total duration to send all data.
 if sys.version_info >= (3, 5):
     # the timeout to connect is 20 seconds, and the read timeout is 2000 seconds
     # the 2000 seconds was calculated with: 100MB (max block size)/ 50KB/s (an arbitrarily chosen minimum upload speed)
-    DEFAULT_SOCKET_TIMEOUT = (20, 2000) # type: ignore
+    CONNECTION_TIMEOUT = 20
```
Comment (lmazuel) [inline sdk/storage/azure-storage-blob/azure/storage/blob/_shared/constants.py]: Detail: redondant with line 14

Comment (rakshith91) [reply sdk/storage/azure-storage-blob/azure/storage/blob/_shared/constants.py]: removed

**Pull request merged**